cellprofiler/modules/correctilluminationapply.py

```diff
@@ -1,7 +1,14 @@
 # coding=utf-8
 
 """
+CorrectIlluminationApply
+========================
+
+<<<<<<< HEAD
```

Comment:
@vchernys You'll need to remove lines like these. They indicate merge conflicts:

```
<<<<<<<< HEAD
a version
=======
another version
>>>>>> COMMIT SHA
```